commons/src/test/java/io/hyscale/commons/commandExecutor/TestCommandExecutor.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.commandExecutor;
+
+import io.hyscale.commons.commands.CommandExecutor;
+import io.hyscale.commons.constants.ToolConstants;
+import io.hyscale.commons.models.CommandResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestCommandExecutor {
+    private final String DOCKER_COMMAND = "docker --version";
+    StringBuilder stringBuilder;
+
+    @BeforeEach
+    public void init() {
+       stringBuilder  = new StringBuilder();
+    }
+
+    @Test
+    public void testExecuteAndGetResults() {
```

Comment:
If the test command is very simple we can assert our command output not by just asserting not null on the result